### PR TITLE
fix(hashrate): fix instant hashrate computation

### PR DESCRIPTION
**Pull request by mikevoronov**
This PR contains various of small fixes

### CODE REVIEW

File: docker/Config.default.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ port = "9384"
 # hard-aes = true
 # jit = true
 # secure = false
-argon2 = "default" # possible values are: "ssse3", "avx2" or "default"
+# argon2 = "default" possible values are: "ssse3", "avx2" or "default"
```
Comment (monoid) [inline docker/Config.default.toml:16]: I think a # is needed before "possible".

File: main/examples/on_active_commitment.json [hunk 2]
```diff
@@ -7,8 +7,7 @@
         "difficulty": [0, 1, 20, 46, 211, 35, 45, 244, 66, 39, 119, 16, 52, 187, 177, 135, 126, 192, 43, 227, 206, 249, 254, 64, 220, 76, 59, 103, 226, 253, 139, 187],
         "cu_allocation": {
             "10": [33, 247, 206, 99, 242, 79, 217, 190, 58, 45, 87, 221, 151, 162, 217, 11, 43, 151, 160, 77, 199, 173, 183, 140, 130, 71, 222, 113, 189, 117, 174, 63],
-            "11": [192, 52, 100, 105, 186, 121, 170, 203, 69, 85, 100, 205, 144, 66, 82, 85, 108, 121, 68, 68, 227, 24, 101, 29, 154, 84, 84, 26, 234, 134, 65, 54],
-            "12": [162, 190, 128, 150, 100, 36, 209, 119, 56, 109, 35, 120, 202, 81, 44, 216, 53, 202, 154, 231, 145, 184, 33, 12, 228, 249, 17, 111, 125, 185, 25, 4]
```
Comment (monoid) [inline main/examples/on_active_commitment.json:11]: How strict we are on unrelated changes?

**Review by monoid — APPROVED** [review]
utACK

File: main/examples/on_active_commitment.json (hunk 2 above)
Comment (mikevoronov) [reply main/examples/on_active_commitment.json:11]: I returned it back, but github looks hungs
